pshtt/pshtt.py

```diff
@@ -1134,30 +1134,46 @@ def load_suffix_list():
     return suffixes
 
 
-def initialize_external_data():
+def initialize_external_data(
+    init_preload_list=None,
```

Comment:
Personally I don't love the variable names, but I understand why you did them that way..